src/crawler/crawlers/arso_daily_weather/crawler.js

```diff
@@ -0,0 +1,247 @@
+// imports
+const CrawlerUtils = require('../../utils/crawlerutils');
+const SQLUtils = require('../../utils/sqlutils');
+
+// external imports
+const cheerio = require('cheerio'); // handling DOM
+const moment = require('moment'); // handling time
+
+
+/**
+ * Class for template crawler.
+ */
+class TemplateCrawler {
+    /**
+     * Responsible for loading state
+     */
+    constructor() {
+        // loading config
+        this.config = CrawlerUtils.loadConfig(__dirname);
+        // loading state
+        this.state = CrawlerUtils.loadState(__dirname);
+
+        this.SQLUtils = new SQLUtils();
+    }
+
+    /**
+     * Responsible for crawling at one step.
+     *
+     * NOTE: Take care that the crawler is resistant to delays (it crawls
```

Comment:
You can remove this comment in the instance; instead you could describe how the crawler actually works.